webapp/components/wpt-results.html

```diff
@@ -80,7 +80,7 @@
         class="query"
         placeholder="Search test files, like cors/allow-headers.htm">
       <div class="path">
-        <a href="/" on-click="navigate">WPT</a>
+        <a href="/" onclick="{{ bindNavigate() }}">WPT</a>
```

Comment:
This bit I'll admit to not understanding. Reviewing on phone, so I'll assume it works :)